MonoGame.Framework/Platform/WindowsUniversal/InputEvents.cs

```diff
@@ -35,6 +35,22 @@ internal class KeyChar
         // To convert from DIPs (device independent pixels) to actual screen resolution pixels.
         private static float _currentDipFactor;
 
+        private CoreIndependentInputSource _coreIndependentInputSource;
+
+        /// <summary>
+        /// Sets the cursor of <see cref="CoreIndependentInputSource"/> thread
+        /// </summary>
+        public CoreCursor CoreCursor
+        {
+            set
+            {
+                _coreIndependentInputSource.Dispatcher.TryRunAsync(
```

Comment:
You are never setting `_coreIndependentInputSource`, its always gonna be null?

Reply:
Hi Harry! I'm gonna check to answer you.

Reply:
I made a fix Harry. Sorry for this, I made a mistake with the first pull request, then after creating the new one I lost a piece.